tests/test_pyglmnet.py

```diff
@@ -73,3 +75,20 @@ def test_multinomial_gradient():
     y_pred = glm.predict(X)
     assert_equal(y_pred.shape, (10, 2, 2))  # n_classes x n_samples x n_classes
     assert grad_beta0[0] != grad_beta0[1]
+
+def simple_cv_scorer(obj, X, y):
+    """Simple scorer takes average pseudo-R2 from regularization path"""
+    yhats = obj.predict(X)
+    ynull = np.zeros(y.shape) * y.mean()
+    return np.mean([obj.pseudo_R2(y, yhat, ynull) for yhat in yhats])
+
+def test_cv():
+    """Simple CV check"""
+    X, y = make_regression()
+    model_mn = GLM(distr='normal', alpha=0.01, reg_lambda=np.array([0.0, 0.1, 0.2]))
+    model_mn.fit(X, y)
+
+    cv = KFold(X.shape[0], 5)
+
+    # check that it returns 5 scores
+    assert_equal(len(cross_val_score(model_mn, X, y, cv=cv, scoring=simple_cv_scorer)), 5)
```

Comment:
At least my code is piggybacking on pseudo_R2 which needs a null model. I am not an expert in `poisson` models and I don't know what a simple null model would be there. @pavanramkumar needs to make a decision